operations/loyaltyprograms.md

```diff
@@ -82,6 +86,19 @@ Note this operation uses [Pagination](../guidelines/pagination.md).
 | `Id` | string | required | Unique identifier of the loyalty program. |
 | `Name` | string | required | Name of the loyalty program. |
 | `Code` | string | required | Code of the loyalty program. |
+| `Type` | [Loyalty program type](#loyalty-program-type) | string | optional | Type of the loyalty program. |
+| `Subscription` | [Loyalty program subscription](#loyalty-program-subscription) | string | optional | Subscription of the loyalty program. |
+
+#### Loyalty program type
+
+* `Hotel`
```

Comment:
Why did we decide to name it `Hotel` program and not `Internal` program? Isn't `Internal` more extensible? 🤔

Reply:
Let's make sure we pick the right naming here. Right now it's easy to change it still.

Reply:
cc @mjumews01 🙏

Reply:
We've confirmed with Mathias these are the standards on [a previous PR](https://github.com/MewsSystems/mews/pull/41893#issuecomment-1512821863))

Reply:
very well 👍